src/Raven.Server/Documents/Handlers/Debugging/MemoryStatsHandler.cs

```diff
@@ -134,6 +135,25 @@ public Task MemorySmaps()
             }
         }
 
+        [RavenAction("/admin/debug/memory/query-ws", "GET", AuthorizationStatus.Operator, IsDebugInformationEndpoint = true)]
+        public Task MemoryQueryWs()
+        {
+            if (PlatformDetails.RunningOnLinux == true)
```

Comment:
I don't like this.
Better to have a single endpoint that would use this method and smaps on Linux to get the values in a cross platform manner.